ddd/projet_doctoral/doctorat/epreuve_confirmation/use_case/write/soumettre_epreuve_confirmation.py

```diff
@@ -0,0 +1,69 @@
+# ##############################################################################
```

Comment:
On a l'habitude de suffixer le nom de fichier des use_case par _service 
Et ne pas suffixer le nom de la méthode

J'ai peut-être raté d'autre service avant.

Reply:
Oui merci c'est noté, @SebCorbin m'a fait la même remarque dans une autre PR (https://github.com/uclouvain/osis-admission/pull/133/) dans laquelle j'ai renommé tous les services qui avaient le même problème.